speculos/api/finger.py

```diff
@@ -18,11 +18,13 @@ def post(self):
         action = args["action"]
         delay = args.get("delay", 0.1)
 
+        x1, y1 = args["x"], args["y"]
         if action == "press-and-release":
-            self.seph.handle_finger(args["x"], args["y"], True)
+            self.seph.handle_finger(x1, y1, True)
             self.seph.handle_wait(delay)
-            self.seph.handle_finger(args["x"], args["y"], False)
+            x2, y2 = args.get("x2", x1), args.get("y2", y1)
+            self.seph.handle_finger(x2, y2, False)
```

Comment:
In @abonnaudet-ledger implementation, there was another `handle_finger` at pos x2, y2 with True, before the release. I guess it's not mandatory?

Reply:
:thinking: Indeed there's a double `handle_finger` in the `screen.py`, like a complete press-and-release action.
Locally, on the API front it works with only one.

Reply:
It also seems to work correctly with only one call on `screen.py:ouseReleaseEvent`
 Maybe there's a corner case I don't get @abonnaudet-ledger ?

Reply:
Yes it's not necessary, I can remove it if you want